### DISCUSSION TITLE
How to register H0n3 modules?

### DISCUSSION THREAD

**Discussion by robbox4711** [tellurianinteractive/Tellurian.Trains.ModulesRegistryApp#72] [Q&A] [answered]
I wonder how do you register H0n3 modules?

**Comment by fjallemark** [comment, marked as answer]
There are two things:
- There must be a *module standard* for H0n3. I havde added **FREMO/S-H03** and **MMM/H0n3**. I found the facts at [Svensk MJ-WIKI](http://www.svenskmjwiki.se/H0n3). If you have any corrections to this, submit an issue of what ro be corrected.
- If you also want want to maintain data for other people's H0n3 modules, I can add a group for that, where you become an administrator and can add other people that also have H0n3-modules. Before you add a person, you must have that person's consent for doing so. You will also be able to invite these people to become users of the Module Registry.
